addons/kibana/6.8.x/kibana-1.yaml

```diff
@@ -83,3 +83,14 @@ spec:
       extraVolumeMounts:
         - mountPath: /usr/share/kibana/plugins/
           name: plugins
+      ingress:
```

Comment:
This change needs to be made via a new revision, rather than updating a previous revision.